.spelling

```diff
@@ -26,6 +26,7 @@
 1.8.x
 1.8.x.
 1.9.x
+1.10.x
```

Comment:
This is a double entry (down two lines).

Reply:
thanks, fixed